addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+import base64
+import logging
+import io
+
+from lxml import etree
+from xml.sax.saxutils import escape, quoteattr
+
+from odoo import _, api, fields, models, tools
+from odoo.tools import cleanup_xml_node
+from odoo.tools.pdf import OdooPdfFileReader, OdooPdfFileWriter
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMoveSend(models.Model):
+    _inherit = 'account.move.send'
+
+    enable_ubl_xml = fields.Boolean(compute='_compute_send_mail_extra_fields')
+    checkbox_ubl_xml = fields.Boolean(
+        string="BIS Billing",
+        compute='_compute_checkbox_ubl_xml',
+        store=True,
+        readonly=False,
+    )
+
+    enable_cii_xml = fields.Boolean(compute='_compute_send_mail_extra_fields')
+    checkbox_cii_xml = fields.Boolean(
+        string="CII Billing",
+        compute='_compute_checkbox_cii_xml',
+        store=True,
+        readonly=False,
+    )
+
+    @api.model
+    def _get_default_enable_cii_xml(self, move):
+        return move.journal_id._get_cii_builder() and\
+                not move.invoice_pdf_report_id and\
+                move.journal_id.checkbox_cii_xml
+
+    @api.model
+    def _get_default_enable_ubl_xml(self, move):
+        return move.journal_id._get_ubl_builder() and\
+                not move.invoice_pdf_report_id and\
+                move.journal_id.checkbox_ubl_xml
```

Comment:
You don't need to know they attachment id here, knowing if there is content is enough